neurom/analysis/morphtree.py

```diff
@@ -237,11 +237,11 @@ def get_bounding_box(tree):
     [xmax, ymax, zmax]]
     """
 
-    min_x = min(p[0] for p in val_iter(tr.ipreorder(tree)))
-    min_y = min(p[1] for p in val_iter(tr.ipreorder(tree)))
-    min_z = min(p[2] for p in val_iter(tr.ipreorder(tree)))
-    max_x = max(p[0] for p in val_iter(tr.ipreorder(tree)))
-    max_y = max(p[1] for p in val_iter(tr.ipreorder(tree)))
-    max_z = max(p[2] for p in val_iter(tr.ipreorder(tree)))
+    min_x = min(p[COLS.X] for p in val_iter(tr.ipreorder(tree)))
```

Comment:
Ah, that is lida's code, BTW, and not sure it will be used now that I have a neuron-wide bounding box.